src/sinks/gcp/stackdriver_logs.rs

```diff
@@ -173,8 +181,16 @@ impl HttpSink for StackdriverSink {
         let log = event.into_log();
 
         let mut entry = map::Map::with_capacity(3);
+        let mut entry = map::Map::with_capacity(4);
```

Comment:
Did you mean to add this line instead of updating the one before it?

Reply:
Should resolve the check failures above